notebooks/01_structured.ipynb

```diff
@@ -71,7 +71,8 @@
     "from sklearn.tree import export_graphviz\n",
```

Comment:
<p>do you need that import?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>which one? <code>pandas</code>? </p>